docs/Project-Configuration/Project-Commands.md

```diff
@@ -586,10 +586,6 @@ EC2 Parameters:
 
 -   `ami` - For an `ec2` provider, the AMI to start. Must set `ami` or `distro`
     but must not set both.
--   `aws_access_key_id` - AWS access key ID. May set to use a
```

Comment:
Do we have follow up work for notifying people who have these removed fields in their existing `host.create` commands that they won't do anything anymore?

Reply:
If someone were using it, it would already be broken since we don't respect these fields—if they are set, we always overwrite them with the `mci` creds on the app servers. No one has been using it for at least 4+ years, so we are good to safely remove it as a feature of the command without notifying anyone.